packages/mco/components/modals/app-manage-policies/assign-policy-view.tsx

```diff
@@ -141,7 +128,7 @@ export const AssignPolicyView: React.FC<AssignPolicyViewProps> = ({
     };
     // assign DRPolicy
     const promises = assignPromises(state.policy);
-    Promise.all(promises)
+    await Promise.all(promises)
```

Comment:
not needed... still not blocking the PR for this small nit...
```suggestion
    Promise.all(promises)
```